processing/data_collection/gazette/spiders/sp_itu.py

```diff
@@ -0,0 +1,53 @@
+import dateparser
+import chompjs
+import base64
+
+from datetime import datetime
+from scrapy import Request, Spider
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
```

Comment:
```suggestion
import base64
from datetime import datetime

import chompjs
import dateparser
from scrapy import Request, Spider

from gazette.items import Gazette
from gazette.spiders.base import BaseGazetteSpider
```

Just to follow PEP8's import rule.